internal/daemon/controller/handlers/hosts/host_service.go

```diff
@@ -643,15 +643,16 @@ func validateCreateRequest(req *pbs.CreateHostRequest) error {
 					len(attrs.GetAddress().GetValue()) < static.MinHostAddressLength ||
 					len(attrs.GetAddress().GetValue()) > static.MaxHostAddressLength {
 					badFields["attributes.address"] = fmt.Sprintf("Address length must be between %d and %d characters.", static.MinHostAddressLength, static.MaxHostAddressLength)
-				}
-				_, _, err := net.SplitHostPort(attrs.GetAddress().GetValue())
-				switch {
-				case err == nil:
-					badFields["attributes.address"] = "Address for static hosts does not support a port."
-				case strings.Contains(err.Error(), "missing port in address"):
-					// Bare hostname, which we want
-				default:
-					badFields["attributes.address"] = fmt.Sprintf("Error parsing address: %v.", err)
+				} else {
+					_, _, err := net.SplitHostPort(attrs.GetAddress().GetValue())
+					switch {
+					case err == nil:
+						badFields["attributes.address"] = "Address for static hosts does not support a port."
+					case strings.Contains(err.Error(), "missing port in address"):
```

Comment:
perhaps we can define a const for this important error msg as well.  It's used like 8 times on our code base.  Even a helper func that returns if it's actual this error like: `func missingPortError(err) bool`.  It's just an idea.

Reply:
The target package has this constant, I don't know if we need to define it more centrally than that, but it could be a constant in this package too.

Reply:
✅